packages/yosqlite/yosqlite.0.1/opam

```diff
@@ -0,0 +1,43 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Simple Sqlite3. Yojson+Sqlite3 amalgamation"
```

Comment:
I'm not totally clear how to parse this synopsis. Is the `.` meant to behere?

Reply:
No need to change this if it may be a reading comprehension issue on my part, btw. :)

Reply:
Yep, it might be confusing. I'm changing the synopsis and description.